resources/user.rb

```diff
@@ -41,7 +41,13 @@
     recursive true
   end
 
-  execute "execute[sensuctl user create #{new_resource.name} -p #{new_resource.password} -r #{new_resource.roles}]" do
+  file object_file do
```

Comment:
as this contains a password you should use `sensitive: true` to prevent it from being displayed in the logs.